### PR TITLE
Spark Riak TS Parquet example added

### PR DESCRIPTION
**Pull request by tmatvienko**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -140,6 +140,37 @@ bin/run-example parquet.ScalaRiakParquetExample
...
+Run it locally:
+```
+bin/run-example streaming.StreamingTSExample
+```
+This will start Spark streaming job waiting for your messages. Message format should be the same as in the example below:
+```
+{"time": "2016-01-01 08:30:00.000", "weather": "sunny", "temperature": 25.0, "humidity": 67.0, "pressure": 30.20, "family": "f"}
+```
+
+Sources [StreamingKVExample.scala](./src/main/scala/com/basho/riak/spark/examples/streaming/StreamingKVExample.scala)
```
Comment (mdigan) [inline examples/README.md]: I think it should be StreamingTSExample.scala here instead of StreamingKVExample.scala

**Comment by mdigan** [comment]
Let's remove the section on the Interactive Scala Shell from the README (https://github.com/basho/spark-riak-connector/tree/parquet-example/examples#interactive-scala-shell). We don't explain it or use it for the examples at all so I don't think it's necessary to include it. 

cc @srgg

File: examples/README.md (hunk 1 above)
Comment (srgg) [inline examples/README.md]: @mdigan  This example should be updated according to the latest changes for Streaming writes.

Comment (mdigan) [inline examples/README.md]: @srgg I just created a JIRA issue about updating the example with the latest changes for Streaming support.

**Comment by korry8911** [comment]
I removed the section on using jupyter notebook @mdigan @srgg

**Pull request merged**